hbw/config/yml_api.default.yml

```diff
@@ -983,6 +1036,38 @@ post:
             type: "select"
             value: null
             scope: "global"
+      - id: "111112"
+        url: "http://bpm.example.com/activiti-rest/service/runtime/tasks/815194"
```

Comment:
http://www.yaml.org/spec/current.html#anchor/information model
Repeated nodes are first identified by an anchor (marked with the ampersand - “&”), and are then aliased (referenced with an asterisk - “*”) thereafter.